app/static/css/cursors.css

```diff
@@ -15,7 +15,8 @@
 }
 
 *[cursor="dot"] > * {
-  cursor: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAA4AAAAOCAYAAAAfSC3RAAAAb0lEQVQ4T9WSQQ4AMQRFuZfju5cJKcG0adNduyOer3yE/5NJTlOY8yUAABGZc4hWGvUZDIiZiygRWZxhBw3qQB9ZGzis4FLpCNypeRNXDcWHQP3Dbty+VWWuzuELuzJAwEO+nHAcfWm5XNwN2z0NHyeAUAvGKOgOAAAAAElFTkSuQmCC"),
+  cursor: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAA4AAAAOCAYAAAAfSC3RAAAAb0lEQVQ4T9WSQQ4AMQRFuZfju5cJKcG0adNduyOer3yE/5NJTlOY8yUAABGZc4hWGvUZDIiZiygRWZxhBw3qQB9ZGzis4FLpCNypeRNXDcWHQP3Dbty+VWWuzuELuzJAwEO+nHAcfWm5XNwN2z0NHyeAUAvGKOgOAAAAAElFTkSuQmCC")
+      6 6,
```

Comment:
Is this fixing alignment? It seems to just be a change in whitespace that doesn't affect functionality.

Reply:
Yes the 6 6 is the alignment it was left out when refactored

Reply:
Oh, I see. I didn't realize `6 6` wasn't in the first line. I'm not sure where it got dropped, but it was gone before the refactor:

https://github.com/mtlynch/tinypilot/blob/a2006b9f03e34acec79885f044825a3cb3b6f51e/app/static/css/style.css#L97

Reply:
yea hmm I know I had put it in there when you mentioned that the cursor wasn't aligned. Not sure what happened. Oh well, there it is now :)